engine/fprintd.go

```
@@ -0,0 +1,155 @@
+const (
+	fprintBusName       = "net.reactivated.Fprint"
+	fprintManagerPath   = dbus.ObjectPath("/net/reactivated/Fprint/Manager")
+	fprintManagerIface  = "net.reactivated.Fprint.Manager"
+	fprintDeviceIface   = "net.reactivated.Fprint.Device"
+	fprintVerifyTimeout = 15 * time.Second
+)
+
+// fprintdAvailable returns nil if the fprintd service is active and has at
+// least one usable device. It is used by `omaseal doctor` to report hardware
+// availability.
+func fprintdAvailable(ctx context.Context) error {
```

golangci-lint (2.13.2)

[error] func fprintdAvailable is unused

(unused)